app/controllers/actions/AuthenticatedControllerComponents.scala

```diff
@@ -52,6 +52,9 @@ trait AuthenticatedControllerComponents extends MessagesControllerComponents {
       getData
   }
 
+  def authAndGetDataAndCheckVerifyEmail(): ActionBuilder[AuthenticatedDataRequest, AnyContent] =
```

Comment:
Not sure what to do here. Logic will be done in a later story

Reply:
Yeah that'll be implemented in the email verification ticket so you can remove this for now.